cli/lib/util.js

```diff
@@ -241,6 +241,14 @@ const parseOpts = (opts) => {
 
   debug('parsed cli options %o', cleanOpts)
 
+  if (cleanOpts.component && cleanOpts.e2e) {
+    util.logErrorExit1(new Error(stripIndent`
+      Error: You passed both --component and --e2e, but only one can be provided.
+
+      See https://docs.cypress.io/ for more information on testing types.
```

Comment:
This is the same link Launchpad uses when you click "need help" on the "learn more about testing types" dialog. Hopefully we'll have a better documentation link at some point, but for now this was the best I could find.

Reply:
Needs to be an on link. I don't think this doc exists, but they should be creating 'on' links so that they can later link them to a generated doc. I'll talk to the CT team about this. 

```suggestion
      See https://on.cypress.io/ for more information on testing types.
```

Reply:
Looks like the launchpad isn't quite as consistent with this as it should be:

![image](https://user-images.githubusercontent.com/3003404/140981012-6ea4aded-e43c-4a64-b965-c3e918ef0c45.png)